interface/main/onotes/office_comments_full.php

```diff
@@ -108,11 +109,14 @@
 $result_count = 0;
 foreach ($result as $iter) {
     $result_count++;
-    
+
+    $date=date( "Y-m-d" ,strtotime($iter{"date"}));
+    $date=oeFormatShortDate($date);
+
     if (getdate() == strtotime($iter{"date"})) {
-        $date_string = "Today, " . date( "D F dS" ,strtotime($iter{"date"}));
+        $date_string = xlt("Today, ") . $date;
```

Comment:
1. Not html escaping here, so used xl()
2. Constants should not include trailing or leading spacing/punctuation except when end of sentence.
``````
$date_string = xl("Today") . ", " . $date;